community/unofficial-download.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Unofficial Downloads
+parent: Community
+permalink: /community/unofficial-downloads/
+---
+
+Requirements for being listed as an unofficial download of FreeTube:
+
+- The download must have been updated in the last month.
```

Comment:
Maybe change it to "The download must be updated within one month of a FreeTube release"